.github/workflows/osrm-backend-docker.yml

```diff
@@ -0,0 +1,30 @@
+name: build and publish container image
+on:
+  create:
+    tags:
+      - v*
+
+jobs:
+  publish:
+    runs-on: ubuntu-latest
+    steps:       
+      - name: Check out the repo
+        uses: actions/checkout@v2
+
+
+      - name: Log in to GitHub Docker Registry
+        uses: docker/login-action@v1
+        with:
+          registry: ghcr.io
+          username: ${{ github.actor }}
+          password: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Build container image
+        uses: docker/build-push-action@v2
+        with:
+          push: true
+          file: ./docker/Dockerfile
+          tags: ghcr.io/${{ github.repository }}/osrm-backend:${{ GITHUB_REF#refs/*/ }} 
```

Comment:
Have you considered using [docker/metadata-action](https://github.com/docker/metadata-action)?
If has some nice Docker tag extraction functionality (e.g. we can push `v5.26.0`, and `latest` on the same container).

Reply:
docker/metadata-action seems nice - but I can't figure out how to combine that with the normal/debug/assertions variations